src/roles/vnsutil-predeploy/tasks/vcenter.yml

```diff
@@ -22,6 +22,7 @@
       tasks_from: configure-ovftool
     vars:
       management_network: "Management"
+      vm_folder: "{{ vcenter.vm_folder | default(omit) }}"
```

Comment:
Your changes to configure-ovftool don't use this variable. configure-ovftool uses `vcenter.vmfolder` and ignores this variable. This is not needed. It isn't being used.